packages/db/config/configUtils.ts

```diff
@@ -0,0 +1,91 @@
+import { eq } from "drizzle-orm";
+
+import { db } from "../index";
+import { config } from "../schema";
+import { ConfigType, ConfigTypeMap, ConfigValue } from "./configValue";
+
+/**
+ * @returns the value only from the database without taking the value from the environment into consideration. Undefined if it does not exist
+ */
+async function getConfigValueFromDB<T extends ConfigType>(
+  configValue: ConfigValue<T>,
+): Promise<ConfigTypeMap[T] | undefined> {
+  try {
+    const rows = await db
+      .select()
+      .from(config)
+      .where(eq(config.key, configValue.key));
+    if (rows.length === 0) {
+      return void 0;
+    }
+    return parseValue(configValue, rows[0].value);
+  } catch (e) {
+    return void 0;
+  }
+}
+
+/**
+ * @returns the value from the environment variable. Undefined if it does not exist.
+ */
+export function getConfigValueFromEnv<T extends ConfigType>(
+  configValue: ConfigValue<T>,
+): ConfigTypeMap[T] | undefined {
+  const environmentValue = process.env[configValue.key];
+  if (!environmentValue) {

```

Comment:
You should check explicitly for undefined here. Empty strings can be a value env variable value.
```suggestion
  if (environmentValue === undefined) {

```